test-fixtures/worker-thread/src/multi.server.env.ts

```diff
@@ -0,0 +1,33 @@
+import { workerThreadInitializer } from '@wixc3/engine-runtime-node';
+import multiFeature, { serverEnv, workerEnv } from './multi.feature';
+
+const workersCount = 3;
+
+multiFeature.setup(serverEnv, ({ run, multiWorkerEcho, workerResponseConfig }, { COM: { communication } }) => {
+    run(async () => {
+        const workerEnvs = await Promise.all(
+            new Array(workersCount).fill(undefined).map(() =>
+                workerThreadInitializer({
+                    communication,
+                    env: workerEnv,
+                })
+            )
+        );
+
+        let exitCode = 0;
+        for (const workerEnv of workerEnvs) {
+            const workerEchoService = multiWorkerEcho.get({ id: workerEnv.id });
+            const response = await workerEchoService.ping();
+
+            if (response !== workerResponseConfig.response) {
+                exitCode = 1;
```

Comment:
so if the next worker succeeds, this gets overridden?

Reply:
no, this overrides with '1' only if response differs from expected one